client/src/controllers/ProgressController.ts

```diff
@@ -25,15 +25,15 @@ export class ProgressController extends Controller {
   static targets = ['label'];
   static values = {
     active: { default: '', type: String },
-    durationSeconds: { default: 30, type: Number },
+    duration: { default: 10000, type: Number },
```

Comment:
As per comment in Slack, the default should be 30 seconds, this is the existing 'production' release's default.

https://github.com/wagtail/wagtail/blob/f99d493af4c1a2f9e0417a6207a9b396fffe7056/client/src/entrypoints/admin/core.js#L310-L314